### PR TITLE
Add blog fellowship of teachers

### PR DESCRIPTION
**Pull request by emilytraff**
New blog for some DM activity starting w/c 17th Jan.

An email will include a link to this post. Please could this be live as soon as possible next week so everything can be put in place. Thanks!

### CODE REVIEW

**Comment by peteryates** [comment]
Looks great overall @emilytraff - I think it just needs a footer paragraph as it's styling the final one (an answer) as a footer. Everything else is spot on though

Screenshot from 2022-01-06 14-01-15

**Comment by emilytraff** [comment]
Thanks @peteryates - I've added a footer!

File: app/views/content/blog/the-fellowship-of-teachers.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ 
+## What have you learnt from your colleagues?
+
+Mike: My teaching assistant suggested giving out different badges at the end of each day to reward kids for showing characteristics like respect or self-belief or honesty. They love it – they’ve really latched onto the idea!
+ 
+## Dare we ask, who’s stricter?
+
+Mike: I think Phil is probably more vigilant than me…
+
+Phil: Underneath it I’m a big softie though!
```
Comment (EllieNodder) [inline app/views/content/blog/the-fellowship-of-teachers.md:53]:
```suggestion
Phil: Underneath it, I’m a big softie though!
```

File: app/views/content/blog/the-fellowship-of-teachers.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+Mike: Lunch!! No, just kidding…. Every day is different, so I never know what’s really going to happen, that’s what makes it great. 
+ 
+## What have you learnt from your colleagues?
+
+Mike: My teaching assistant suggested giving out different badges at the end of each day to reward kids for showing characteristics like respect or self-belief or honesty. They love it – they’ve really latched onto the idea!
+ 
+## Dare we ask, who’s stricter?
+
+Mike: I think Phil is probably more vigilant than me…
```
Comment (EllieNodder) [inline app/views/content/blog/the-fellowship-of-teachers.md:51]:
```suggestion
Mike: I think Phil is probably more vigilant than me …
```

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me!

**Pull request merged**